MapboxNavigation-SPM.xcodeproj/project.pbxproj

```diff
@@ -73,6 +73,7 @@
 		C53F2F0720EBC95600D9798F /* Example.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = Example.app; sourceTree = BUILT_PRODUCTS_DIR; };
 		C57AD98920EAA39A0087B24B /* Entitlements.plist */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.xml; path = Entitlements.plist; sourceTree = "<group>"; };
 		C5D9800C1EFA8BA9006DBF2E /* CustomViewController.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; name = CustomViewController.swift; path = Example/CustomViewController.swift; sourceTree = "<group>"; };
+		D27E7C5F26E9E72300CA5926 /* 212_3.gpx */ = {isa = PBXFileReference; lastKnownFileType = text.xml; path = 212_3.gpx; sourceTree = "<group>"; };
```

Comment:
This file isn’t included in the PR, so we’ll need to leave it out of the project.